content/_data/events/2019-06-30-beijing-jam.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "JAM in Beijing"
+date: "2019-06-30T13:00:00"
+link: "https://www.meetup.com/Beijing-Jenkins-Area-Meetup/events/262169759/"
+---
+Jenkins Area Meetup in Beijing.
```

Comment:
Would be nice to have some info about the agenda

Reply:
OK. I'm not the host but still will do that.